python-package/lightgbm/plotting.py

```diff
@@ -487,9 +497,20 @@ def create_tree_digraph(booster, tree_index=0, show_info=None, precision=3,
         The index of a target tree to convert.
     show_info : list of strings or None, optional (default=None)
         What information should be shown in nodes.
-        Possible values of list items:
-        'split_gain', 'internal_value', 'internal_count', 'internal_weight',
-        'leaf_count', 'leaf_weight', 'data_percentage'.
+
+            - ``'split_gain'`` : gain from adding this split to the model
+            - ``'internal_value'`` : raw predicted value that would be produced \
+                by this node if it was a leaf node
+            - ``'internal_count'`` : number of records from the training data \
+                that fall into this non-leaf node
+            - ``'internal_weight'`` : total weight of all nodes that \
+                fall into this non-leaf node
+            - ``'leaf_count'`` : number of records from the training data \
+                that fall into this leaf node
+            - ``'leaf_weight'`` : total weight of all nodes that \
+                fall into this leaf node
+            - ``'data_percentage'`` : percentage of training data that fall into \
+                this node
```

Comment:
FYI, we don't have line length limit for Python files. So next time there is no need to split such short lines in "unexpected" places. 🙂 

```suggestion
            - ``'split_gain'`` : gain from adding this split to the model
            - ``'internal_value'`` : raw predicted value that would be produced
              by this node if it was a leaf node
            - ``'internal_count'`` : number of records from the training data
              that fall into this non-leaf node
            - ``'internal_weight'`` : total weight of all nodes that
              fall into this non-leaf node
            - ``'leaf_count'`` : number of records from the training data
              that fall into this leaf node
            - ``'leaf_weight'`` : total weight of all nodes that
              fall into this leaf node
            - ``'data_percentage'`` : percentage of training data that fall into
              this node
```

Reply:
oh ok, I didn't realize. I'll just change them to single lines right now in this PR.

I originally had this written like your suggestion in https://github.com/microsoft/LightGBM/pull/3618/files#r534262617, but that causes sphinx to shade the first line gray.